cmd/kubectl-testkube/commands/artifacts.go

```diff
@@ -67,9 +69,9 @@ func NewListArtifactsCmd() *cobra.Command {
 	return cmd
 }
 
-func NewDownloadArtifactsCmd() *cobra.Command {
+func NewDownloadSingleArtifactsCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "download",
+		Use:   "download-one",
```

Comment:
shouldn't we handle this with flags like "download --all" instead of replacing it with "download-one" subcommand?

Reply:
don't think so - branching code is worse than splitting it

Reply:
we'll see what users say but I think that most use cases should be for downloading all artifacts - passing those names and escaping them when you'll have spaces will be pain in the ass.